keanu-project/src/main/java/io/improbable/keanu/model/regression/RegressionGraph.java

```diff
@@ -13,7 +13,7 @@
 import lombok.Getter;
 import lombok.Value;
 
-public class LinearRegressionGraph<OUTPUT> implements ModelGraph<DoubleTensor, OUTPUT> {
+public class RegressionGraph<OUTPUT> implements ModelGraph<DoubleTensor, OUTPUT> {
```

Comment:
I think `LinearRegressionGraph` is the better name. Logistic Regression is still linear so it's appropriate to use a LinearRegressionGraph in both models.